pulsar-broker/src/test/java/org/apache/pulsar/broker/resourcegroup/RGUsageMTAggrWaitForAllMesgsTest.java

```diff
@@ -57,7 +57,7 @@
 // The tenants and namespaces in those topics are associated with a set of resource-groups (RGs).
 // After sending/receiving all the messages, traffic usage statistics, and Prometheus-metrics
 // are verified on the RGs.
-public class RGUsageMTAggrWaitForAllMesgs extends ProducerConsumerBase {
+public class RGUsageMTAggrWaitForAllMesgsTest extends ProducerConsumerBase {
```

Comment:
You need to assign these to one of the existing test groups used amoung broker tests.

Reply:
AFAICT, it runs under the group "other".